godot-core/src/builtin/vectors/vector3.rs

```diff
@@ -325,6 +325,12 @@ mod test {
         );
     }
 
+    #[test]
+    fn sign() {
+        let vector = Vector3::new(0.2, -0.5, 0.0);
+        assert_eq_approx!(vector.sign(), Vector3::new(1., -1., 0.));
```

Comment:
This could be `assert_eq!`, no? Since the numbers can be precisely represented...

In other places too